k8s/services/imagepullsecrets/helm-0.3.cue

```diff
@@ -0,0 +1,77 @@
+package imagepullsecrets
+
+#KubeVersion: [=~"^0\\.3\\."]: minor: >=21
+#Values: [=~"^0\\.3\\."]: {
+	replicas: int | *1
+	istio: {
+		revision: ""
+	}
+	podAnnotations: {}
+	podSecurityContext: {
+		runAsNonRoot: bool | *true
+		seccompProfile: {
+			type: "RuntimeDefault"
+		}
+	}
+	securityContext: {
+		allowPrivilegeEscalation: bool | *false
+		capabilities: {
+			drop: ["ALL"]
+		}
+	}
+	image: {
+		repository: "ghcr.io/banzaicloud/imagepullsecrets"
```

Comment:
same issue, it should be a schema and a default value `string | *"ghcr.io/banzaicloud/imagepullsecrets"`